### PR TITLE
add gsss + some clean up

### PR DESCRIPTION
**Pull request by LarryMatte**

### CODE REVIEW

File: src/main/java/com/arcbees/bourseje/client/resources/AnimationsResources.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * the License.
+ */
+
+package com.arcbees.bourseje.client.resources;
+
+import com.arcbees.gsss.animation.client.AnimationResources;
+import com.google.gwt.resources.client.ClientBundle;
+
+public interface AnimationsResources extends ClientBundle {
+
```
Comment (christiangoudreau) [inline src/main/java/com/arcbees/bourseje/client/resources/AnimationsResources.java]: rm 23

File: src/main/java/com/arcbees/bourseje/client/resources/FontResources.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+import com.google.gwt.resources.client.ClientBundle;
+import com.google.gwt.resources.client.DataResource;
+
+public interface FontResources extends ClientBundle {
+    String MIME_TYPE_TTF = "application/font-sfnt";
+    String MIME_TYPE_EOT = "application/vnd.ms-fontobject";
+    String MIME_TYPE_WOFF = "application/font-woff";
+    String MIME_TYPE_SVG = "image/svg+xml";
+
```
Comment (christiangoudreau) [inline src/main/java/com/arcbees/bourseje/client/resources/FontResources.java]: rm 27

**Comment by christiangoudreau** [comment]
LGTM, why the build is failing?

**Comment by meriouma** [comment]
Add `<extraJvmArgs>-Duser.language=en -Duser.country=US</extraJvmArgs>` to the gwt-maven-plugin configuration

File: src/main/resources/com/arcbees/bourseje/client/resources/css/styles.gss [hunk 3]
```diff
@@ -59,7 +28,7 @@ html, body {
 
 body {
     font-size: 15px;
-    font-family: 'brandon_grotesque_regularRg';
+    font-family: 'brandonGrotesqueReg';
```
Comment (jasonlemay) [inline src/main/resources/com/arcbees/bourseje/client/resources/css/styles.gss]: should define a default after, like `Arial, sans-serif`

**Pull request merged**